geo/map_processing/graph.py

```diff
@@ -38,30 +33,43 @@ def __init__(self, parent):
         self.streets = []
         self.level = 0
         if self.parent:
-            self.level = self.parent.level+1
+            self.level = self.parent.level + 1
 
 
 class Graph:
+
     def __init__(self):
         self.faces = tuple([MapNode(None) for x in range(0, NUM_FACES)])
 
-    def get_cell_neighbors(self, cell):
+    def get_cell_neighbors(self, cell: S2Cell) -> list:
```

Comment:
Use Sequence[s2.S2Cell] as the return type instead of 'list'.

(You'll need to do "from typing import Sequence)

Reply:
done for all